ansible/01_prepare_host.yaml

```diff
@@ -242,23 +196,6 @@
         /usr/sbin/xfs_growfs /
       when: lv_resized is changed
 
-  - name: Enable RH CSB repository for CA cert
-    copy:
-      dest: /etc/yum.repos.d/rhel8-csb.repo
-      content: |
-        [rhel8-csb]
-        name=RHEL8 CSB packages
-        baseurl=http://hdn.corp.redhat.com/rhel8-csb
```

Comment:
thought we need that to be able to verify the gitlab https cert, but seems not as its already cloned in a previous task :)

Reply:
This might actually be a bug which is masked by another issue.

Note that we use the git ansible action before we install git? In my testing I noticed that this block was being skipped. I didn't investigate further because everything was working, but it looks like this is because I don't have secrets_repo installed. It's quite possible that this cert is actually required and it just didn't come up.

However, that also leaves us with a bootstrapping problem: how do we install git before pulling the secret from git which allows us to install git?